paasta_tools/chronos_tools.py

```diff
@@ -394,6 +395,15 @@ def get_healthcheck_mode(self, _):
         # Healthchecks are not supported yet in chronos
         return None
 
+    def get_desired_state_human(self):
```

Comment:
if you have the marathon-version of ``get_desired_state_human`` in marathon_tools and a chronos-version in chronos_tools, then the caller will have to make a decision on which to call. This is fine for now because the caller is in marathon_serviceinit and chronos_serviceinit, so you'll have to update each of those to call ``chronos_tools.get_desired_state_human`` and ``marathon_tools.get_desired_state_human`` respectively.